sys/amd64/conf/MINIMAL

```diff
@@ -78,6 +78,15 @@ options 	INCLUDE_CONFIG_FILE	# Include this file in kernel
 # Debugging support.  Always need this:
 options 	KDB			# Enable kernel debugger support.
 options 	KDB_TRACE		# Print a stack trace for a panic.
+# For full debugger support use (turn off in stable branch):
+include ../../conf/std.debug
+
+# Kernel Sanitizers
+#options 	COVERAGE		# Generic kernel coverage. Used by KCOV
+#options 	KCOV			# Kernel Coverage Sanitizer
+# Warning: KUBSAN can result in a kernel too large for loader to load
+#options 	KUBSAN			# Kernel Undefined Behavior Sanitizer
+#options 	KCSAN			# Kernel Concurrency Sanitizer
```

Comment:
Why add these lines? Why not KASAN and KMSAN too?

This looks like it belongs in a different commit.

Reply:
> Why add these lines? Why not KASAN and KMSAN too?

this is what's currently in `GENERIC`, i just copied it as-is.  ASAN and KMSAN are only referenced in their dedicated config files `GENERIC-K?SAN`.  curiously, `CSAN` is mentioned in both places

this might be a good time to make this more consistent; does it make sense to just document all the *SAN options in GENERIC/MINIMAL?

Reply:
Ah, I see. Would it make more sense to have it in std.debug, so that they're not duplicated?

Reply:
yes, but since users can't (or shouldn't) edit std.debug just to build a new kernel, i think this should come with a comment in GENERIC referencing the file and suggesting copying the lines from there to the appropriate kernel config -- does that seem reasonable?

Reply:
Well, given that we already have GENERIC-KASAN etc., this stuff probably really belongs in sys/amd64/conf/NOTES.

Reply:
i've moved these to NOTES for amd64.  other archs might need similar changes, i'll check those too.